### DISCUSSION TITLE
How to change expiry interval in emqx dashboard for a client

### DISCUSSION THREAD

**Discussion by karanbangia1998** [emqx/emqx#10153] [Q&A] [answered]
Our frontend client is using courier library with `isCleanSession = false` in emqx dashboard, we are able to see expiry interval of 7200s which is leading to message getting delivered even after client reconnects, is there any way to reduce this time?
![Screenshot 2023-03-16 at 3 32 05 PM](https://user-images.githubusercontent.com/95840238/225583146-74e98927-560d-4fd5-b992-060a93749bce.png)

Its mentioned that expiry time is session exipry time [here](https://www.emqx.io/docs/en/v4.4/getting-started/dashboard.html#dashboard-interface:~:text=Expiry%20Interval,interval%2C%20unit%3A%20seconds)

**Comment by lafirest** [comment, marked as answer]
@karanbangia1998 
I think this https://www.emqx.com/en/blog/mqtt5-new-feature-clean-start-and-session-expiry-interval might help you.
And what you need to modify maybe is this one `mqtt.session_expiry_interval`, you can see here for details, https://www.emqx.io/docs/en/v5.0/configuration/configuration-manual.html#mqtt-basic-parameters

**Reply by thalesmg** [reply]
You may find that in the dashboard under Configuration > MQTT > Session.

![2023-03-21_14-03](https://user-images.githubusercontent.com/16166434/226686647-b61b0447-659a-48d5-9aa1-e86f75df0915.png)

**Comment by lafirest** [comment]
But I don't know what mean of `message getting delivered even after client reconnects`, I think this is the right behavior
